scene/resources/external_texture.cpp

```diff
@@ -30,7 +30,6 @@
 
 #include "external_texture.h"
 
-#include "drivers/gles3/storage/texture_storage.h"
```

Comment:
This include isn't used anywhere here (no GLES3 specific code here) so not sure why it was included, can't test on Android to confirm that this won't break the functionality there but I don't see how it could